course_discovery/apps/course_metadata/views.py

```diff
@@ -25,16 +26,14 @@ def post(self, request):
             )
 
         try:
-            # One site, One Partner/Org:
-            # Get a `class Partner` instance by the `request.site.domain`(e.g., 0.0.0.0:18000)
-            course_site_partner = Partner.objects.get(site__domain=request.site.domain)
+            partner = getattr(settings, 'PARTNER', None)
             access_token, __ = EdxRestApiClient.get_oauth_access_token(
-                '{root}/access_token'.format(root=course_site_partner.oidc_url_root.strip('/')),
-                course_site_partner.oidc_key, course_site_partner.oidc_secret, token_type='JWT'
+                '{root}/access_token'.format(root=partner['OIDC_URL_ROOT'].strip('/')),
+                partner['OIDC_KEY'], partner['OIDC_SECRET'], token_type='JWT'
             )
             kwargs = {
-                'course_key': target_course_id, 'partner': course_site_partner,
-                'api_url': course_site_partner.courses_api_url,
+                'course_key': target_course_id, 'partner': partner,
+                'api_url': partner['COURSES_API_URL'],
```

Comment:
- This function of class `CourseMetadataRefresher` is triggered when course content get changed from platform and Refresh course metadata in Course Discovery Service.